docs/tutorials/BuildMultiClusterApplication/BuildMultiClusterApplication.md

```diff
@@ -63,19 +65,21 @@ spec:
     overrideRules:
       - clusterSelector:
           clusterSelectorTerms:
-            - labelSelector:
-                matchLabels:
-                  cluster-name: member-1
+            - labelSelector: {} # select all the clusters
         jsonPatchOverrides:
           - op: add
             path: /metadata/annotations
             value:
-              {"service.beta.kubernetes.io/azure-dns-label-name":"fleet-multi-cluster-app-member-1"}
+              {"service.beta.kubernetes.io/azure-dns-label-name":"multi-cluster-app-${MEMBER-CLUSTER-NAME}"}

```

Comment:
nit: Link to the resource override doc explaining how ${MEMBER-CLUSTER-NAME} works would be helpful